index.html

```diff
@@ -1480,133 +1480,175 @@ <h3>The <dfn data-dfn-for="">Observable</dfn> interface</h3>
-          <span class="rfc2119-assertion" id="scripting-sec-update">
-              Therefore, if WoT Runtime supports post-manufacturing provisioning or update of WoT scripts, WoT
-              runtime or any related data, such operations SHOULD be done in a secure fashion.
-          </span>
-          A set of recommendations for secure update and post-manufacturing provisioning can be found in
-          [[!WOT-SECURITY-BEST-PRACTICES]].
-      </li>
-      <li>Typically the WoT runtime needs to store the security credentials that are provisioned to a WoT
-      device to operate in WoT network. The confidentiality or integrity of these credentials should not
-      be compromised.
-          <span class="rfc2119-assertion" id="scripting-sec-creds1">
-              Therefore the WoT runtime SHOULD securely store the provisioned security credentials,
-              guaranteeing their integrity and confidentiality.
-          </span>
-          <span class="rfc2119-assertion" id="scripting-sec-creds2">
-            In case there are more than one tenant on a single WoT-enabled device, a WoT Runtime SHOULD
-            guarantee isolation of each tenant provisioned security credentials.
-          </span>
-          Additionally, in order to minimize a risk that provisioned security credentials get compromised,
-          the WoT runtime should not have any way for WoT scripts to query these credentials.
-          <span class="rfc2119-assertion" id="scripting-sec-creds3">
-            Therefore, the WoT Runtime SHOULD NOT expose any API for WoT scripts to query the provisioned
-            security credentials.
-          </span>
-      </li>
-   </ul>
+ <section id="sec-security-consideration-script"> <h3>Scripts Security and Privacy Risks</h3>
+  <p>This section contains specific risks relevant for script developers.
   </p>
 
- <p>Some additional specific recommendations relevant for WoT script developers:
-  <ul>
-        <li>A typical way to compromise any process is to send it a corrupted input via one of the exposed
-           interfaces.
-          <span class="rfc2119-assertion" id="scripting-sec-inputs">
-           Therefore developers SHOULD perform validation on all WoT script inputs, including
-           <a href="https://en.wikipedia.org/wiki/Fuzzing">fuzzing</a>.
-          </span>
-            There are many tool and techniques in existence to do such validation.
-            More details can be found in [[!WOT-SECURITY-TESTING]].
+  <section id="sec-security-consideration-script-input">
+   <h5>Corrupted Input Security and Privacy Risk</h5>
+   <p>
+      A typical way to compromise any process is to send it a corrupted input via one of the exposed
+      interfaces. This can be done to a script instance using WoT interface it exposes.
+    </p>
+    <dl><dt>Mitigation:</dt><dd>
+        Developers should perform validation on all script inputs.
+        In addition to input validation,
```

Comment:
This is fine to mention, but we'd need to group separately which statements target 1) API implementors, 2) script developers. 

The Scripting API spec primarily targets the API implementors (using the Runtime) and at least some of the security statements should be normative text.

The recommendations for developers should be informative text and would be preferably grouped separately.

Reply:
OK, let's discuss this at the next's week security call, if you can make it?

Reply:
Or it might be simpler to merge this PR and then I can modify it. This way all versions will be recorded in history and you lose less time shuffling things around following other people's ideas. 
Then of course we can discuss the modified version PR in the next security call.

Reply:
That would certainly work for me! Thank you Zoltan!